config/routes.rb

```diff
@@ -29,4 +29,9 @@
   resource :user_onboardings, only: :update
 
   resources :current_optional_subjects, only: :index
+  resources :reviews, only: [:destroy] do
+    collection do
+      post :upsert
+    end
+  end
```

Comment:
```suggestion
  resources :reviews, only: [:create, :destroy]
```

i would use the standard `create` action (and still do the upsert)

Reply:
👉 [Rename reviews upsert endpoint](https://github.com/cedarcode/mi_carrera/pull/597/commits/8680464038c312afafae12de9caf5d9f5bc305aa)